packages/firestore/test/integration/api/query.test.ts

```diff
@@ -555,6 +557,58 @@ apiDescribe('Queries', persistence => {
     });
   });
 
+  it('can use IN filters', async () => {
```

Comment:
Per the conversation at standup, we'll need to figure out a way to make these only run against the emulator for now (e.g. check that we're running in node and the emulator environment variable is set).